movie-browser/src/components/FeatureMovie/FeatureMovie.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const FeatureMovies = () => {
```

Comment:
you can also make it a one liner, since you're just returning and not doing any logic inside this function (react component) you can easily just return it, like so:

```
const FeatureMovies = () => <div>Display Feature Movie Section</div>;
```

Reply:
understand what you mean